src/config/swaggerDoc.js

```diff
@@ -30,7 +30,7 @@ const swaggerDefinition = {
 
 const options = {
   swaggerDefinition,
-  apis: ['src/routes/*.js'],
+  apis: ['src/routes/api/*.js'],
```

Comment:
Try adding a second value to generate the docs for the index routes

`src/routes/*js`